CMakePresets.json

```diff
@@ -0,0 +1,20 @@
+{

```

Comment:
If you want these new files in the releases please add to `Makefile.am` after `EXTRA_DIST+= CMakeLists.txt` at line 158.

Reply:
done